snap/snapcraft.yaml

```diff
@@ -221,6 +221,17 @@ parts:
     - -DMULTIPASS_UPSTREAM=origin
     - -DMULTIPASS_ENABLE_FLUTTER_GUI=on
     override-build: |
+      # Install Rust stable toolchain
+      if [ ! -f "$HOME/.cargo/env" ]; then
+        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable
+        source $HOME/.cargo/env
+      fi
+      # Ensure Rust is in PATH for this build
+      export PATH="$HOME/.cargo/bin:$PATH"
+
+      # Run the rust tests
+      (cd ${CRAFT_PART_SRC}/crates && cargo test --all)
```

Comment:
We're already running the tests in a dedicated CI step, right? So this being here would mean running the Rust tests twice.